R/update.R

```diff
@@ -308,8 +309,77 @@ setMethod("update", "parameter_list", function(object,.y,...) {
   object
 })
 
-##' @export
-##' @rdname update
-setMethod("update", "ev", function(object,y,...) {
-  
-})
+
+#' Update parameters, initials, and settings within a model object
+#' 
+#' The main use case for using [within] rather than [update] or [param] or
+#' [init] is when you want to update to a new value that is calculated from 
+#' the existing value.  See the example in details
+#' 
+#' Other model object slots that can be updated: `start`, `end`, `delta`, 
+#' `add`, `rtol`, `atol`, `hmax`, `maxsteps`.  These are include for convenience, 
+#' but we expect that most of the time these will get updated through the 
+#' update method.
+#' 
+#' @param data an object with class mrgmod
+#' @param expr expressions evaluated in an environment containing various model
+#' object components, including parameters, initial conditions, and others 
+#' (see details)
+#' @param ... not used
```

Comment:
I think it's required here:

>A method must have all the arguments of the generic, including … if the generic does.

https://cran.r-project.org/doc/manuals/R-exts.html#Generic-functions-and-methods

Will take it out if it can be taken out.